Source/NetworkActivityIndicatorManager.swift

```diff
@@ -172,29 +195,29 @@ public class NetworkActivityIndicatorManager {
 
     // MARK: - Private - Notification Registration
 
-    private func registerForNotifications() {
+    public func registerForNotifications(
+        requestDidStartNotificationNames: [Notification.Name] = defaultRequestDidStartNotificationNames,
```

Comment:
This function works particularly well with AFNetworking; you can hook this library up to AFNetworking in one call.